tools/ci/Jenkinsfile

```diff
@@ -23,6 +23,19 @@ node("Component") {
     }
 
     image.inside {
+        stage("Build") {
```

Comment:
Build natively or something like this for clarity

Reply:
Change to "Build for Host".